app-text/calibre/files/calibre-5.44.0-Fix-compatibility-with-zeroconf-0.73.patch

```diff
@@ -0,0 +1,26 @@
+From 5d8d85c649b181747d182f676cfd76f843bd61f0 Mon Sep 17 00:00:00 2001
```

Comment:
Link to the upstream PR/commit at the top? (See https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto too)

Reply:
Erm, that would be upstream commit 5d8d85c649b181747d182f676cfd76f843bd61f0 as included. :D

Is a hyperlink necessary?

Reply:
I'd like it if we do it in future, because it's clear signal "yes, this is upstream, and you can click it to find out a bit more" (including the tags it might be in, etc), as opposed to "this is something I grabbed from a PR/mailing list/private email and it never got merged". But that's a principle thing -- in this case, you're free to leave it as-is.